website/files/models/base.py

```diff
@@ -573,10 +586,10 @@ def touch(self, auth_header, revision=None, **kwargs):
             self.generate_waterbutler_url(revision=revision, meta=True, **kwargs),
             headers=headers,
         )
+        # Returns None to addon_view_or_download_file 'version'
```

Comment:
Removed. That was just a note to myself while I was trying to figure out the code. I missed it when I was cleaning up.